src/fox_lab_to_nwb/streets_conversion.py

```diff
@@ -0,0 +1,133 @@
+from pathlib import Path
+import time
+from datetime import datetime
+from typing import Optional
+from zoneinfo import ZoneInfo
+
+import parse
+from neuroconv.utils import dict_deep_update, load_dict_from_file
+from neuroconv import ConverterPipe
+from neuroconv.datainterfaces import DeepLabCutInterface, VideoInterface
+
+from fox_lab_to_nwb.behavior import BehaviorInterface
+
+
+def run_trial_conversion(trial_data_folder: Path, output_dir_path: Optional[Path] = None, verbose: bool = True):
+
+    if verbose:
+        start_time = time.time()
+
+    if output_dir_path is None:
+        output_dir_path = Path.home() / "conversion_nwb"
+
+    # Parse metadta from the trial_data_folder name
+    # Define the correct parsing pattern
+    pattern = r"{cross}_{date}_{time}_f{fly_number}_r{trial_repeat_number}"
```

Comment:
```suggestion
    pattern = r"{cross}_{date:%y%m%d}_{time}_f{fly_number}_r{trial_repeat_number}"
```

you can do this if it' helpful